data-model/fdo-types/0.3.0/digital-media-objects/schema/digital-entity.json

```diff
@@ -0,0 +1,182 @@
+{
+  "$id": "https://schemas.dissco.tech/schemas/fdo-types/0.3.0/digital-media-objects/digital-entity.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$comment": "DigitalObject Version 0.3.0",
+  "type": "object",
+  "properties": {
+    "@id": {
+      "type": "string",
+      "description": "The unique identifier of the object",
+      "pattern": "https:\\/\\/doi\\.org\\/10\\.\\d+\\/[-\\w]+",
+      "examples": [
+        "https://doi.org/10.22/XXX-XXX-XXX"
+      ]
+    },
+    "@type": {
+      "type": "string",
+      "description": "The type of the object",
+      "const": "ods:DigitalEntity"
+    },
+    "ods:ID": {
```

Comment:
isn't this covered by @id?

Reply:
Yes was doubting there. Might still be good to have a term where we can put it. The term we can include in the vocabulary, so a bit redundant but probably more clear. And when we do non-jsonld we can just exclude these properties.